ner/src/main/java/edu/illinois/cs/cogcomp/ner/ExpressiveFeatures/BrownClusters.java

```diff
@@ -7,22 +7,30 @@
  */
 package edu.illinois.cs.cogcomp.ner.ExpressiveFeatures;
 
+import cogcomp.Datastore;
+import cogcomp.DatastoreException;
+import edu.illinois.cs.cogcomp.core.resources.ResourceConfigurator;
 import edu.illinois.cs.cogcomp.ner.IO.InFile;
 import edu.illinois.cs.cogcomp.ner.IO.ResourceUtilities;
 import edu.illinois.cs.cogcomp.ner.LbjTagger.Data;
 import edu.illinois.cs.cogcomp.ner.LbjTagger.NEWord;
 import edu.illinois.cs.cogcomp.ner.LbjTagger.ParametersForLbjCode;
 import edu.illinois.cs.cogcomp.lbjava.parse.LinkedVector;
 import gnu.trove.map.hash.THashMap;
+import io.minio.errors.InvalidEndpointException;
+import io.minio.errors.InvalidPortException;
```

Comment:
If you want to access minio namespace here, it is recommended to add that as a dependency to ner/pom.xml